gazeplay-games/src/main/java/net/gazeplay/games/mediaPlayer/MediaFileReader.java

```diff
@@ -26,17 +26,19 @@
     @Getter
     private final List<MediaFile> mediaList;
 
-    private int index;
+    @Getter
+    private SimpleIntegerProperty firstMediaDisplayedIndex;
 
     @Getter
-    @Setter
-    private int playing;
+    private SimpleIntegerProperty playingMediaIndex;
 
     MediaFileReader(IGameContext gameContext) {
         this.gameContext = gameContext;
         mediaList = new ArrayList<>();
-        index = -1;
-        playing = -1;
+        firstMediaDisplayedIndex = new SimpleIntegerProperty();
+        firstMediaDisplayedIndex.setValue(-1);
+        playingMediaIndex = new SimpleIntegerProperty();
+        playingMediaIndex.setValue(-1);
```

Comment:
You can set the default value in the constructor of the `SimpleIntegerProperty`.
```suggestion
        playingMediaIndex = new SimpleIntegerProperty(-1);
```